backend/theme/serializers.py

```diff
@@ -20,8 +20,11 @@ def serialize_header(header: HeaderConfig) -> dict:
     return {
         'nextExperimentButtonText': _('Next experiment'),
         'aboutButtonText': _('About us'),
-        'showScore': header.show_score
-    }
+        'showScore': header.show_score,
+        'scoreClass': 'gold',
+        'scoreLabel': _('Points'),
+        'noScoreLabel': _('No points yet!')
```

Comment:
Thinking out loud... Maybe it'd be nice to put this in a `score` dictionary? Something like:

```suggestion
        'score': {
            'scoreClass': 'gold',
            'scoreLabel': _('Points'),
            'noScoreLabel': _('No points yet!')
            }
```

Or thinking even louder: don't pass the `showScore` boolean. Simply define the score object if `showScore` is true and otherwise set it to `None`. The frontend can interpret the `None` (`null` in js) as not having to show the score.

```suggestion
        'score': {
            'scoreClass': 'gold',
            'scoreLabel': _('Points'),
            'noScoreLabel': _('No points yet!')
            } if header.show_score else None,
```

Reply:
I agree that having the information in its own dictionary is more readable, testable, and portable.